core.js

```diff
@@ -0,0 +1,197 @@
+    })
+    BetterWeather.currentQuery = outputHandler.join(' ');
+    $('#location-input').val('');
+  },
+
+  accessCurrentLocation(){
+    $('.location-button').click(()=>{
+      navigator.geolocation.getCurrentPosition(
+        (position) =>{ 
+          BetterWeather.currentLocation = {
+            lat: position.coords.latitude,
+            long: position.coords.longitude
+          }
+          BetterWeather.searchCurrentLocation();
+        },
+
+        ()=>{
+          $('.location-button').html('location disabled').removeClass('btn-success').addClass('btn-danger').off();
+        }
+      );
+    })
+  },
+
+  searchCurrentLocation(){
+    $.ajax({
+      method: "GET",
+      url: `https://api.openweathermap.org/data/2.5/find?lat=${BetterWeather.currentLocation.lat}&lon=${BetterWeather.currentLocation.long}&cnt=1&appid=0cc115d6b4190cb95525f9c0f6f2b58c&units=imperial`,
+      dataType: "json",
+      success(data){
+        BetterWeather.currentQuery = data.list[0].name;
+        BetterWeather.currentWeatherSearch();
+        BetterWeather.forecastSearch();
+      },
+      error(_jqXHR, textStatus, errorThrown){
+        console.error(textStatus);
+        console.error(errorThrown);
+      },
+   })
+  },
+
+  currentWeatherSearch(){
+    $.ajax({
+      method: "GET",
+      url: `https://api.openweathermap.org/data/2.5/weather?q=${BetterWeather.currentQuery}&appid=0cc115d6b4190cb95525f9c0f6f2b58c&units=imperial`,
+      dataType: "json",
+      success(data){
+        BetterWeather.renderWeather(data);
+      },
+      error(_jqXHR, textStatus, errorThrown){
+        BetterWeather.errorMessage(errorThrown);
+      },
+   })
+  },
+
+  renderWeather(data){
+    $('.weather-container').empty();
+    const source = $('#weather-template').html();
+    const template = Handlebars.compile(source);
+    const newItem = template({
+      temp: Math.round(data.main.temp),
+      city: data.name,
+      weather: data.weather[0].main,
+      icon: data.weather[0].icon
+    })
+    $('.weather-container').append(newItem);
+    BetterWeather.checkCurrent();
+    backgrounds.renderStyles(data.weather[0].main);
+    BetterWeather.renderMaps();
+  },
+
+  renderMaps(){
+    const source = $('#map-template').html();
+    const template = Handlebars.compile(source);
+    const newMap = template({location: BetterWeather.currentQuery});
+    $('.map-container').append(newMap);
+  },
+
+  forecastSearch(){
+    $.ajax({
+      method: "GET",
+      url: `https://api.openweathermap.org/data/2.5/forecast?q=${BetterWeather.currentQuery}&appid=0cc115d6b4190cb95525f9c0f6f2b58c&units=imperial`,
+      dataType: "json",
+      success(data){
+        BetterWeather.forecast(data);
+        },
+      error(_jqXHR, textStatus, errorThrown){
+        console.error(textStatus);
+        console.error(errorThrown);
+      },
+    })
+  },
+
+  forecast(data){
+    const fiveDay = data.list.filter(element=> element.dt_txt.includes('12:00:00'));
```

Comment:
So only 12:00:00? This is clever, but obviously not ideal. Perhaps a comment here would be helpful.